src/components/Pokemon.js

```diff
@@ -0,0 +1,55 @@
+import React, {useEffect, useState} from 'react';
+import './Pokemon.css';
+import axios from "axios";
+
+function Pokemon({endpoint}) {
+    const [pokemon, setPokemon] = useState({});
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const {data} = await axios.get(endpoint);
+                console.log({data});
+                setPokemon(data);
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        if (pokemon) {
+            fetchData();
+        }
+    }, [endpoint]);
+
+
+    // The Object.keys() method returns an array of a given object's own enumerable string-keyed property names.
+    //https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
+
+    return (
+        <div className="poke-card">
+            {Object.keys(pokemon).length > 0 &&
```

Comment:
Als je de default waarde hebt aangepast, dan kan het hier ook wat simpeler
```suggestion
            {pokemon &&
```